src/main/java/com/icthh/xm/tmf/ms/communication/messaging/handler/MobileAppMessageHandler.java

```diff
@@ -16,7 +16,7 @@
 
 @Service
 @RequiredArgsConstructor
-@LepService
+@LepService(group = "service.message")
```

Comment:
this can broke existing leps (if that exists)

Reply:
Correct. 
I missed _group_ parameter in the initial impl and now want to move the corresponding LEP file to an appropriate place.
I've updated file location on our side.
Please let me know if it's okay with you.

Reply:
Are you sure that no one has managed to write lep-s during this time?